masterlist.yaml

```
@@ -11297,7 +11297,7 @@ plugins:
         condition: 'active("Relationship Dialogue Overhaul.esp") and not active("SDA RDO Patch.esp")'
       - <<: *patchProvided
         subs: [ 'Serana Dialogue Edit' ]
-        condition: 'active("Serana Dialogue Edit.esp") and not active("SDA_SDE Patch.esp")'
+        condition: 'active("Serana Dialogue Edit.esp") and not active("SDA SDE Patch.esp")'
```

We can check for both, in case anyone wants to use the old version or renames it.
```suggestion
        condition: 'active("Serana Dialogue Edit.esp") and not active("SDA SDE Patch.esp") and not active("SDA_SDE Patch.esp")'
```